tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/decoders/navigate-to-decoders.when.js

```diff
@@ -1,19 +1,14 @@
 import { When } from 'cypress-cucumber-preprocessor/steps';
-import { getObject } from '../../../utils/driver';
+import { clickElement } from '../../../utils/driver';
 import {
-  menuListButtonsSelector,
-  subMenuListButtonsSelector,
-  wazuhButtonSelector,
+  decodersButton,
+  wazuhButton,
+  managementButton,
 } from '../../../pageobjects/wazuh-menu/wazuh-menu.page';
 
 When('The user navigates to decoders', () => {
-  getObject(wazuhButtonSelector)
-    .click();
-  getObject(menuListButtonsSelector)
-    .eq(0)
```

Comment:
nice approach removing the ".eq(#)" and handling everything on the selector

Reply:
Thanks!!